app/assets/stylesheets/custom.css.scss

```diff
@@ -133,6 +133,13 @@ a:hover, a:active {
     margin-bottom: 20px;
 }
 
+.btn-details {
+    background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces